inconspiquous/dialects/qft.py

```diff
@@ -0,0 +1,40 @@
+from xdsl.dialects.builtin import IntAttr
+from xdsl.ir import Dialect
+from xdsl.irdl import irdl_attr_definition, ParameterDef
+from xdsl.parser import AttrParser
+from xdsl.printer import Printer
+
+from inconspiquous.gates import GateAttr
+
+
+@irdl_attr_definition
+class QFTAttr(GateAttr):
+    name = "qft.n"
+
+    num_qubits_param: ParameterDef[IntAttr]
```

Comment:
```suggestion
    qubits: ParameterDef[IntAttr]
```
Would this be simpler?